js/ui/endSessionDialog.js

```diff
@@ -417,6 +447,18 @@ var EndSessionDialog = new Lang.Class({
         this._user.disconnect(this._userChangedId);
     },
 
+    _syncUpdateStatus: function()  {
```

Comment:
Hm, now that I'm looking at it again, I believe `_syncAutomaticUpdateStatus` is a better name, even if more verbose. This dialog already has to deal with updates coming from PackageKit, so `_syncUpdateStatus` is ambiguous in this context.